spec.html

```diff
@@ -1157,6 +1157,18 @@ <h1>The String Type</h1>
         *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_"*.<br>
         For historical reasons, it has significance to various algorithms.
       </p>
+      <p>
+        The <dfn>ASCII-uppercase</dfn> of a String value _S_ is the String value derived from _S_ by replacing each occurrence of an ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) with the corresponding ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) while preserving all other code units.
+        The ASCII-uppercase of a sequence of Unicode code points _A_ is the sequence of code points derived from _A_ by replacing each occurrence of an ASCII lowercase letter code point (U+0061 through U+007A, inclusive) with the corresponding ASCII uppercase letter code point (U+0041 through U+005A, inclusive) while preserving all other code points.
+      </p>
+      <p>
+        The <dfn>ASCII-lowercase</dfn> of a String value _S_ is the String value derived from _S_ by replacing each occurrence of an ASCII uppercase letter code unit (0x0041 through 0x005A, inclusive) with the corresponding ASCII lowercase letter code unit (0x0061 through 0x007A, inclusive) while preserving all other code units.
```

Comment:
402 uses both ASCII-uppercase and ASCII-lowercase terms. If we removed one of them, it'd mean that 402 needs to supply the other one out of context to its sibling. Is this OK? Or should we retain both for downstream use in 402?

I didn't see any 402 usage of the "sequence of Unicode code points" text, so I'll remove it.

Reply:
After discussion on the editors' call, we decided to remove this text completely and move it into the 262 portion of Temporal until Temporal Stage 4 when it will come back!  The related sentence in the Time Zone Identifiers section is also removed in the latest commit.